### DISCUSSION TITLE
release version scheme

### DISCUSSION THREAD

**Discussion by olumolu** [Jeffser/Alpaca#178] [General]
X.Y.Z
Lets say the variable x can be chnaged for major upgrade or chnages in the app such as a 35-<.. commits.
Y variable for 15-25
For Z 3-15

## don't use terms such as small update various fix many fixes and some feature update.
This thing can be in the description rather just add the version number that is all.

![Screenshot_20240801-220847~2](https://github.com/user-attachments/assets/7af15a4c-a217-48b0-8c53-1b98669825c8)
![Screenshot_20240801-220815~2](https://github.com/user-attachments/assets/212ec8da-6aca-4b04-a84e-251952a41f6f)
![Screenshot_20240801-220707~2](https://github.com/user-attachments/assets/d791a665-9211-403c-a927-a39b76c9286e)

**Comment by Jeffser** [comment]
yeah maybe but I do like the funny release titles like 'another small update', I think it makes the app development more personal

**Comment by olumolu** [comment]
Change release cycle and design
Ad whenever you want to publish a new version obviously there are some update for translation and stuff sometime none but
I would recommend to initiate discussion and ask translators for update so it can be done and publish a full fledged translated app. As it was but due to your release translator don't know the update of .pot so they can't update them.
As you update pot and publish with in a 30min span.

**Reply by Jeffser** [reply]
I don't want to bother translators asking them to translate the app before the newer version comes out, I also don't want to wait for every translation to get done before releasing a new version. Everyone has their own schedules and I don't expect my app to have up to date translations all the time

**Reply by olumolu** [reply]
Understand your point.
